Containers/apache/Caddyfile

```diff
@@ -4,6 +4,9 @@
     storage file_system {
         root /mnt/data/caddy
     }
+
+    # See https://github.com/nextcloud/all-in-one/issues/828
+    # trusted_proxies placeholder
```

Comment:
![grafik](https://user-images.githubusercontent.com/75573284/173842620-19e3b82e-7fdf-470e-87a6-90d2a695916c.png)
I think this does not work here

Reply:
I see, that is unfortune but okay if it works

Reply:
hm.... this is getting worse then I thought since we need brackets now which are hard to parse and modify via sed...

Reply:
okay, seems like empty brackets are still accepted so not as problematic as I thought.